prediction_market_agent/db/models.py

```diff
@@ -6,6 +6,7 @@
 
 class LongTermMemories(SQLModel, table=True):
     __tablename__ = "long_term_memories"
+    __table_args__ = {"extend_existing": True}
```

Comment:
Fixes `sqlalchemy.exc.InvalidRequestError: Table 'long_term_memories' is already defined for this MetaData instance.`. Is it a proper fix?